packages/marko-web-theme-default/components/site-header/index.marko

```diff
@@ -16,6 +16,7 @@ $ const navigation = {
   modifiers=input.modifiers
   attrs=input.attrs
```

Comment:
Add this tag to the Marko json file

Also, you might as well add a below nav tag while you’re in here